src/app/auth/confirm.html

```diff
@@ -0,0 +1,13 @@
+<div layout="vertical" layout-fill>
+    <md-content layout-fill>
+        <header>
+            <navbar></navbar>
+        </header>
+
+        <div layout="row" layout-align="center center" class='md-display-1' md-whiteframe="24" style="margin-top: 124px">

```

Comment:
Not sure if this really needs its own page?  Could we just do a popup and redirect the user to login?